.github/workflows/ci.yaml

```diff
@@ -18,7 +18,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [ 1.20.x, 1.21.x, 1.22.x ]
+        go-version:
+          - 1.20.x
```

Comment:
Remove 1.20? I think that's what we've done in other repos. We generally support back to N-2 (one farther back than the Go team).

Reply:
We don't really need to ratchet it forward to 1.21, but I can.

Reply:
I need to disable the 1.20 required check first, so will merge this as-is and follow up with a PR to remove it.